src/main/resources/swagger/api-docs.yaml

```diff
@@ -3638,7 +3638,51 @@ paths:
         414:
           description: |
             URI length exceeds the configured limit of 2048 characters.
-            Please use the POST endpoint when it's necessary to supply a large number of attribute names.
+            When necessary to supply a large number of attribute names, please use one of:
+            * POST /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/tsv
+            * POST /cookie-authed/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/tsv
+          content: {}
+        500:
+          description: Internal Server Error
+          content: {}
+      x-passthrough: false
+    post:
```

Comment:
this `POST` endpoint is the new download-to-bucket endpoint. It lives at `/api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/tsv`. So, it's parallel to the existing download tsv API:
* GET: download to browser
* POST: save to bucket